lib/Service/Appointments/BookingCalendarWriter.php

```diff
@@ -103,8 +103,8 @@ public function write(AppointmentConfig $config,
 		if (!($calendar instanceof ICreateFromString)) {
 			throw new RuntimeException('Could not find a public writable calendar for this principal');
 		}
-
-		$organizer = $this->userManager->get($config->getUserId());
+		$organizerId = $config->getUserId();
+		$organizer = $this->userManager->get($organizerId);
```

Comment:
Refactoring here doesn't make sense as the user object is required anyway. It should only be refactored when fetching the whole user object just to call `getDisplayName()` and nothing else.